src/content/changelogs/pages.yaml

```
@@ -5,6 +5,15 @@ productLink: "/pages/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-12-19"
+    title: Cloudflare GitHub App Permissions Update
+    description: |-
+      - Users with the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed to connect repositories to Pages or Workers will be prompted to update permissions **today**. New permissions will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you. 
+      - **Requested permissions:** 
+        - [Repository Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration) (read/write) to create repositories
+        - [Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents) (read/write) to push code to the created repositories 
+      - **Action:** When you receive the email from GitHub with the subject “[GitHub] Cloudflare Workers & Pages is requesting updated permission”, please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.* 
```

pet peeve of mine these macos quotes
```suggestion
      - **Action:** When you receive the email from GitHub with the subject "[GitHub] Cloudflare Workers & Pages is requesting updated permission", please review and accept the new permissions. *If you take no action or choose to decline, you can continue to connect repositories and deploy changes via the Cloudflare GitHub App as you do today.* 
```